client/reader/following-manage/subscriptions.jsx

```diff
@@ -14,14 +14,40 @@ import ReaderExportButton from 'blocks/reader-export-button';
 import SitesWindowScroller from './sites-window-scroller';
 import QueryReaderFollows from 'components/data/query-reader-follows';
 import FollowingManageSearchFollowed from './search-followed';
+import { getFeed as getReaderFeed } from 'state/reader/feeds/selectors';
+import { getSite as getReaderSite } from 'state/reader/sites/selectors';
+import { getReaderFollows } from 'state/selectors';
+import UrlSearch from 'lib/url-search';
+import { getSiteName, getSiteUrl, getSiteDescription, getSiteAuthorName } from 'reader/get-helpers';
 
 class FollowingManageSubscriptions extends Component {
 	static propTypes = {
 		follows: PropTypes.array.isRequired,
+		doSearch: PropTypes.func.isRequired,
 	};
 
+	filterFollowsByQuery( query ) {
+		const { getFeed, getSite, follows } = this.props;
+
+		return follows.filter( follow => {
+			const feed = getFeed( follow.feed_ID ); // todo grab feed and site for current sub
+			const site = getSite( follow.site_ID );
+			const phraseRe = new RegExp( escapeRegexp( query ), 'i' );
```

Comment:
might pull this out of the closure to avoid recreating the regexp for every item